src/main/resources/terraform-provider-twilio/api.mustache

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 {{#resources}}
+{{^readOnly}}
```

Comment:
I'd prefer logic in the generator that removes these resources. Then we don't need to delete the files later.

Reply:
Is this still WIP?

Reply:
Yes, but not on this file. I no longer think the logic should be in the template.

Reply:
That is also my preference. I'm exploring using the `.openapi-generator-ignore` [feature](https://openapi-generator.tech/docs/faq-extending/#how-do-i-skip-files-during-code-generation).

Reply:
These are the options so far:
1. post-process the files as demonstrated in this PR
2. pre-process the files and write a list of files to ignore to `.openapi-generator-ignore`
3. dig deeper on how the `openapi-generator` determines what resources to process and filter there (already spent a good amount of time researching this with no solution)

Reply:
You can process the OpenAPI doc beforehand and remove any operations that correspond to a resource without all the CRUD operations. Example of how to process the doc: https://github.com/twilio/twilio-oai-generator/blob/1fdd14860564bb962a3c9870d10916df1a34c36d/src/main/java/com/twilio/oai/TwilioNodeGenerator.java#L56